doc/man5/flux-config-bootstrap.rst

```diff
@@ -61,6 +63,35 @@ hosts
    substitutions work here as well.
 
 
+ZEROMQ ENDPOINTS
+================
+
+In this context, ZeroMQ endpoint URIs normally use the :man7:`zmq_tcp`
```

Comment:
Did you mean `:linux:man7:` (flux-core doesn't have a `zmq_tcp(7)` manpage AFAICS)

Reply:
Just caught that and forced a push - you were too quick! :-)

Although I note that :linux: doesn't have the zmq pages, so the link is broken in the readthedocs preview.  Better than hitting the flux-undocumented page tough I guess.

Reply:
Ah, sorry I thought I had deleted that comment but it must have been cached somehow.

I wonder if it is our choice of "linux" manpage provider. `zmq_tcp(7)` does seem to be available [here](https://linux.die.net/man/7/zmq_tcp)

Reply:
I'll try changing it and see how that goes, thanks!